maven-modules/plugin-management/submodule-1/src/test/java/com/baeldung/CopiesAdditionalResourcesUnitTest.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CopiesAdditionalResourcesUnitTest {
+
+    @Test
+    public void givenAdditionalResource_whenCopyingFromSourceToDestination_thenShouldBeInDestination() {
```

Comment:
As we're using `JUnit5`, by convention the `class` and `void` method do not need the `public` modifier. They can just be package-private.